controllers/utils/peer_ref.go

```diff
@@ -33,3 +33,10 @@ func GetPeerRefForSpokeCluster(mp *multiclusterv1alpha1.MirrorPeer, spokeCluster
 	}
 	return nil, fmt.Errorf("PeerRef for cluster %s under mirrorpeer %s not found", spokeClusterName, mp.Name)
 }
+
+func IsStorageClientType(peerRefs []multiclusterv1alpha1.PeerRef) bool {
+	if peerRefs[0].StorageClusterRef.Namespace == "" && peerRefs[1].StorageClusterRef.Namespace == "" {
+		return true
+	}
+	return false
```

Comment:
Should we use Namespace empty as the condition to verify if it is a StorageClient or check for its existence?
We could simplify this to
```
return peerRefs[0].StorageClusterRef.Namespace == "" && peerRefs[1].StorageClusterRef.Namespace == "" 
```

Reply:
This is just a MirrorPeer check to see if its client pairing . Later more changes will made to API to make it more explicit.